uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -69,4 +70,22 @@ protected function cleanup() {
     }
   }
 
+  private function loadRequiredData() {
```

Comment:
In what situation this data won't be available on the database? I'm asking this because the tests pass on my local after a db schema reset.